IFC 4.3/ProjectSetup_1/UT_ProjectSetup_1.ifc

```diff
@@ -0,0 +1,48 @@
+ISO-10303-21;
+
+/* ------- This file was produced manually for IFC Infra Extension Deployment project. --------- */
+/* ------- The file should be come with a README.md with supplementing info. ------------------- */
+/* ------- Stefan Jaud, stefan.jaud@tum.de, 2020.07.15. ---------------------------------------- */
+
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition [NotDefinedYet]'), '2;1');
+FILE_NAME ('UT_ProjectSetup_1.ifc', '2020-07-15T18:00:00', (''), (''), 'Stefan Jaud manually', 'SublimeText 3.0', '');
+FILE_SCHEMA (('IFC4X3_RC1'));
+ENDSEC;
+
+DATA;
+
+/* ------- General definitions ----------------------------------------------------------------- */
+#1 = IFCDIMENSIONALEXPONENTS(0, 0, 0, 0, 0, 0, 0);
```

Comment:
99% certain you should not have spaces between parameters:
PARAMETER_LIST = PARAMETER { "," PARAMETER }